src/assets/styles/_resilient-green-spaces.scss

```diff
@@ -0,0 +1,149 @@
+.allotments {
```

Comment:
Ditto for this file - I think there is maybe a way to achieve this colour coding without requiring lots of custom css, especially since it looks like there is a lot of duplicated code here (the only differences being the name of the datagroup and the colour).

Reply:
My suggestion would be to add a nullable varchar(7) `colour-hex` column in the data_groups table in the backend, and include this info in the existing datagroups API.

This can then be used directly as the colour value in the front-end, without requiring the logic to match a datagroup title to a colour.

If we wanted to add the same feature for another client, or change colour schemes, we could easily change these values in the database without any code changes.

Reply:
Thanks @rogup that's a great solution. 
I'd be happy with an additional column that indicates the colour (eg 1-5, where the number maps to a hex)
But directly associating a hex is the perfect solution so that we can easily add as many colours as we like :)

Reply:
Thanks for the suggestions and advice @rogup. After I digest and get my head round it all, I might (well most likely 😁) have to give you a shout for some advice.

Reply:
Saying that, I think I have a handle on it. I'll still probably give you a shout tomorrow anyway, @rogup , to run things by you. Or perhaps just drop in my solution in here to get your opinion.

Reply:
@rogup, refactored in line with the above. All working as expected. Nice one, man, this solution is so much cleaner.

Related BE PR also created.
https://github.com/DigitalCommons/land-explorer-back-end/pull/56